client/src/main.ts

```diff
@@ -40,8 +38,4 @@ const vuetify = createVuetify({
   icons
 })
 
-app.use(createPinia())
-app.use(router)
-app.use(vuetify)
-
-app.mount('#app')
+createApp(App).use(createPinia()).use(router).use(vuetify).mount('#app')
```

Comment:
I still can't seem to see the Pinia store in my Vue dev tools. Are there any additional steps that I'm missing?

Reply:
Never mind, I can see it now after main was merged into this branch.